docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>This might be better along with assigning <code>None</code>  / <code>NAN</code>  section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Bumped down the assigning section to just above this section.</p>